gcp/appengine/frontend_handlers.py

```diff
@@ -40,7 +40,9 @@
 _PAGE_SIZE = 16
 _PAGE_LOOKAHEAD = 4
 _REQUESTS_PER_MIN = 30
-_VALID_BLOG_NAME = re.compile(r'^[\w-]+$')
+_WORD_CHARACTERS_OR_DASH = re.compile(r'^[\w-]+$')
+_VALID_BLOG_NAME = _WORD_CHARACTERS_OR_DASH
+_VALID_VULN_ID = _WORD_CHARACTERS_OR_DASH
```

Comment:
You could tighten this regex up a bit more to say `^\w+-[\w-]+$` and be less likely to match on things that aren't actually intended to be vulnerability IDs now, or in the future...

(aside: I can't find any signs of LBSEC IDs, even though they get a mention in the schema docs)

Reply:
Yeah, but it does add a tad bit more complexity (adding a similar but different regex) for little benefit (since we ultimately validate the ID to exist in our DB), and this is a rare edge case that would save us a single Datastore entity read. 

I'll keep this as is in the interests of simplicity.